cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json

```diff
@@ -28,6 +28,7 @@
     "transforms": [],
     "instances": 1
   },
-  "description": "Realtime updates from Amazon Simple Queue Service to HBase."
+  "description": "Real-time updates from Amazon Simple Queue Service into an HBase table",
```

Comment:
"Create a Realtime Pipeline" should be "Create a Real-time Pipeline", but I think that marketing ship has sailed a long time ago...however, I would be happy to change it.